pkg/domain/entities/volumes.go

```diff
@@ -109,6 +110,12 @@ type VolumeInspectReport struct {
 	*VolumeConfigResponse
 }
 
+// VolumePruneOptions describes the options needed
+// to prune a volume from the CLI
+type VolumePruneOptions struct {
+	Filters url.Values `json:"filters" schema:"filters"`
```

Comment:
I was conflicted whether to use `url.Values` or the underlying type of `map[string][]string` for the filters. I saw it both ways throughout the codebase. I settled on `url.Values` but I don't feel strongly either way. I thought I remember seeing some mention of a refactor for filters so hopefully the inconsistency can be fixed then.

Reply:
I feel like we should use `map[string][]string` everywhere.
@baude @jwhonce WDYT?

Reply:
Once @baude / @jwhonce add their input I can fix the other things you mentioned @Luap99 (since the type will effect the changes you requested, even though its not much different).

Reply:
Just keep it for now. It will be reworked soon.